kzz7yh-f42515/cod-ve09v2_kzz7yh-f42515.xml

```diff
@@ -87,9 +87,10 @@
    </p>
             <div xml:id="kzz7yh-f42515-Dd1e147">
                <head xml:id="kzz7yh-f42515-Hd1e149">Quaestio 1</head>
+               <head type="question-title">Utrum omnia luminaria posita sunt in uno corpore continuo</head>
```

Comment:
```suggestion
               <head type="question-title">Utrum omnia luminaria posita sint in uno corpore continuo</head>
```

Reply:
Just wondering - it is sint in the Consequenter section but sunt under Primo ostendo - except both sentences are identical. Why?